specification/sql/resource-manager/Microsoft.Sql/preview/2022-08-01-preview/Servers.json

```diff
@@ -987,6 +988,11 @@
             "name": "ServerNetworkAccessFlag",
             "modelAsString": true
           }
+        },
+        "externalGovernanceEnabled": {
+          "description": "Whether external governance is enabled.",
+          "type": "boolean",
+          "readOnly": true
         }
```

Comment:
Standard guidance is: replace boolean/switch properties with better enum
A boolean will forever have two valid values (true or false). A string enum type is always preferred. Also, properties should always provide better values just than True and False. For example two switches "isTypeA" and "isTypeB" should be replaced with one enum "type": [A, B, DefaultType]. Even if you still believe [True, False] are the correct values for a property, you should use a string enum with values [True, False] instead of boolean. Enums are always a more flexible and future proof option because they allow additional values to be added in the future in a non-breaking way, e.g. [True, False, Unknown].

Reply:
THanks, I have changed this field to an enum.